QuizApp/App/Common/AppDependencies.swift

```diff
@@ -0,0 +1,73 @@
+import Resolver
+import UIKit
+
+extension Resolver: ResolverRegistering {
+
+    public static func registerAllServices() {
```

Comment:
This is one way of doing it. What we usually do on the project is a bit different. Because apps are split into Modules (based on functionality), we have `AppModule` in the main module, and every module has its {X}Module class that conforms to `ModuleProtocol`. AppModule instantiates other modules and registers dependencies for all modules in this container. Here's a short example to make this clearer: 
```

class AppModule {
    private let modules: [ModuleProtocol] = [
        AnalyticsModule(),
        NetworkModule(),
        LoginModule()
     ]

    lazy var container: Resolver = {
        let container = Resolver()
        registerDependencies(in: container)
        return container
    }()
    
    private func registerDependencies(in container: Resolver) {
        modules.forEach { $0.registerDependencies(in: container) }
    }
}
```

```
public class LoginModule: ModuleProtocol {
    public func registerDependencies(in container: Resolver) {
        container
            .register { LoginPresenter(router: container.resolve()) }
            .scope(.unique)

    }

}
```